src/main/java/com/iota/iri/model/persistables/ObsoleteTag.java

```diff
@@ -2,8 +2,18 @@
 
 import com.iota.iri.model.Hash;
 
+/**
+ * This is a collection of hash identifiers that can change the values of an <tt>Obsolete Tag</tt>.
```

Comment:
Consider:
A collection of a {@link TransactionHash} identifiers that are indexed by {@link ObsoleteTagHash} in the data base.

If you don't understand DM me.
I honestly don't understand what you mean by "change".


If you don't i